packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -1096,17 +1123,23 @@ class MemoryBodyState extends State<MemoryBody>
         child: Container(
           padding: const EdgeInsets.fromLTRB(0, 5, 0, 8),
           decoration: BoxDecoration(
-            color: colorScheme.hoverBackgroundColor,
-            border: Border.all(color: Colors.yellow),
+            color: colorScheme.defaultBackgroundColor,
+            border: Border.all(
+                color: Colors.grey[400], width: hover_card_border_width),
```

Comment:
In other places in DevTools, we use `Theme.of(context).focusColor` for border color. Will that work here?

Reply:
Done.